shopinvader_product_sale_packaging/views/product_packaging.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<odoo>
+    <record id="product_packaging_tree_view_inherit" model="ir.ui.view">
+        <field name="model">product.packaging</field>
+        <field
+            name="inherit_id"
+            ref="sell_only_by_packaging.product_packaging_tree_view_inherit"
+        />
+        <field name="arch" type="xml">
+            <field name="force_sale_qty" position="after">
+                <!-- TODO: group on this field was shopinvader.group_shopinvader_manager-->
+                <field name="shopinvader_display" />
```

Comment:
How to manage security now that this module shouldn't depend on `shopinvader_anymore`?

Reply:
Adding "Product / manager" group (base.group_system) for now.